Makefile

```diff
@@ -301,7 +303,7 @@ ifeq ($(CI), true)
 export DOCKER_DEVKIT_AWS_ARGS := --env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY
 endif
 
-ci.e2e.build.all:
+ci.e2e.build.all: flatcar-version.yaml
```

Comment:
Shouldn't be the `flatcar-version.yaml` executed in the devkit container?

Reply:
True. Is there a reason this doesn't just run:
```shell
WHAT="make flatcar" make devkit.run
```

Reply:
Fixed